applications/GeoMechanicsApplication/custom_constitutive/yield_surface.h

```diff
@@ -0,0 +1,29 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Mohamed Nabi
+//                   Wijtze Pieter Kikstra
+//
+
+#pragma once
+
+#include "includes/serializer.h"
```

Comment:
This `#include` is not needed here. Later, when we add the `load` and `save` member functions, we should use a forward declaration. That is, it should be done in the same way as for class `StressStatePolicy`.

However, we do need `#include "includes/ublas_interface.h"`, since the class uses `Vector` as the return type of one of the member functions.

```suggestion
#include "includes/ublas_interface.h"
```